app/views/projects/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h2>Project: <%= @project.name %></h2>
+
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
+<p>Average Contestant Experience: <%= @project.avg_yoe_of_contestants %> years</p>
+
+<h3>Add Contestant</h3>
+<%= form_with url: "/projects/#{@project.id}/contestant", data: {turbo_method: :post} do |form| %>
+  <%= form.label :name  %>
+  <%= form.text_field :name  %>
+  <%= form.label :yoe, "Years of Exp:"  %>
+  <%= form.text_field :yoe  %>
+  <%= form.label :age  %>
+  <%= form.text_field :age  %>
+  <%= form.label :hometown  %>
+  <%= form.text_field :hometown  %>
+
```

Comment:
Let's break this down:
```
I see a form to add a contestant to this project
When I fill out a field with an existing contestants id
And hit "Add Contestant To Project"
```
We are adding an already existing contestant to an already existing project. So your form would only need a `:contestant_id`. Your path looks good. 
```ruby
<%= form_with url: "/project/#{@project.id}/contestants", method: :post, local: true do |form| %>
  <%= form.label :contestant_id %>
  <%= form.number_field :contestant_id %>

  <%= form.submit "Add Contestant To Project" %>
<% end %>
```
You are not creating anything new other that a relationship between an existing contestant and an existing project.